src/homework1.py

```diff
@@ -0,0 +1,388 @@
+import csv
+import doctest
+import glob
+import itertools
+from itertools import groupby
+import random
+import re
+import sys
+from time import localtime
+import unittest
+import xml.etree.ElementTree as etree
+
+# Output
```

Comment:
It looks much more clear if you task description stands closely to task and 
separated from other tasks with two blank lines: 
```python
   pass  # last line of previous task
    
 
   # next task docstring
   pass  # my next task first line  
```